source/components/molecules/GroupedList/GroupedList.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { View, LayoutAnimation } from 'react-native';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import { FieldLabel, Text, Fieldset, Button } from 'source/components/atoms';
+import theme from '../../../styles/theme';
+import { colorPalette } from '../../../styles/palette';
+
+const FieldsetButton = styled(Button)`
+  margin-left: 26px;
+`;
```

Comment:
Will make sure to move this component to be apart of the Fieldset component, since we are using it at several occasions at this point (y)

Reply:
Same implementation but as an export on Fieldset

Reply:
I did this.